### DISCUSSION TITLE
Lets talk about Quests than cant be completed

### DISCUSSION THREAD

**Discussion by cressidagp** [arcemu/arcemu#434] [General]
Hi there comunmity, in the last three years i found many many quests that cant be completed even if i have all necesary things for them.
I never decided to create an issue about this since i was almost sure than this was a ArcDB issue. Recently, i started to doubt about that. Fortunately, discussions are enabled now >D

Here a quick steps to reproduce this:

1. create human character
2. get the quest a "Threat Within" from deputy willem
3. complete quest at marshall mcbride and get quest "Kobold Camp Cleanup"
4. complete quest and get "Ivestigate echo ridge" (15)
5. complete quest and get "Skirmish at Echo Ridge" (21)
6. complete quest and get "Report to Goldshire"
7. recall port goldshire
8. get quest "The Fargodeep Mine" (62)
9. go to explore mine (get sure to get the objetive complete message), and return to goldshire

If plr level >= 6 then: 
	quest ender offers "Westbrook Garrison Needs Help"
	complete button for "The Fargodeep Mine" its unclikable

.modify player level to 5
	quest ender not offers "Westbrook Garrison Needs Help"
	complete button for "The Fargodeep Mine" its clickable

This could be a problem related to 3.3.5, when blizz made quest chain more flexibles and maybe arcemu wasnt prepared for that.

**Comment by dfighter1985** [comment]
Weird. I thought I've already fixed this.
Going to take a look.

**Comment by dfighter1985** [comment]
I can confirm and I've already found a solution.

**Comment by dfighter1985** [comment]
@cressidagp  Check the latest revision.

**Comment by cressidagp** [comment]
It works now! Thank you!!!

pd: looks pretty cool!
